packages/pods-core/src/v2/pods/export/AirtableExportPodV2.ts

```diff
@@ -59,11 +65,28 @@ class AirtableUtilsV2 {
         try {
           const _records = await func(record);
           return _records;
-        } catch (error) {
+        } catch (error: any) {
+          let message;
+          if (error?.statusCode === 422) {
+            const airtableError = error as AirtableError;
+            if (airtableError.error === "INVALID_MULTIPLE_CHOICE_OPTIONS") {
+              // example airtable error message: 'Insufficient permissions to create new select option ""scope.xyz""'
+              const value = airtableError.message.split('""')[1];
```

Comment:
we need to check that [1] index in array actually exists; otherwise let's just print the default error message if it's formatted in an unexpected way.

Reply:
We do have a check for the `field` while creating the message. I updated the message to be the default one in case the value is undefined.
We could just print the default error message but I remember there was a suggestion to include the `field` in the error. For eg. in our case, if the field `status` has value `in-progress`, this error message will let us know that problem lies in the `status` field.

Reply:
> We could just print the default error message but I remember there was a suggestion to include the field in the error.

Right - for the happy case - yes, let's include field. 

But what if airtable decides to reformat their message to say `'Insufficient permissions to create new select option scope.xyz'` without the extra quotation marks?  Then we're going to hit an array out of bounds exception.

Reply:
As discussed offline - should be good.